src/fs/makedev.rs

```diff
@@ -1,5 +1,10 @@
 /// `makedev(maj, min)`
-#[cfg(not(any(target_os = "android", target_os = "emscripten", target_os = "wasi")))]
+#[cfg(not(any(
+    target_os = "android",
+    target_os = "emscripten",
+    target_os = "wasi",
+    target_os = "openbsd"
+)))]
```

Comment:
The existing code admittedly isn't entirely consistent about this, but since the entire `makedev.rs` module is not implemented for openbsd, I'd prefer to put the conditional on the `mod makedev;` line in src/fs/mod.rs, instead of adding conditionals to everything it defines.

Reply:
Ah, you already added the conditions to mod.rs. I have a big PR which conflicts with this one, so I've now taken the liberty of just fixing this, by reverting the changes to makedev.rs, so that I can land this and sort out the merge conflicts on my side.